lib/Ed25519KeyPair.js

```diff
@@ -11,32 +11,72 @@ const LDKeyPair = require('./LDKeyPair');
 
 class Ed25519KeyPair extends LDKeyPair {
   /**
-   * @param options {object} See LDKeyPair constructor docstring for full list
-   *
-   * @param [options.publicKeyBase58] {string}
-   * @param [options.privateKeyBase58] {string}
+   * @classdesc  An implementation of
+   * [Ed25519 Signature 2018]{@link https://w3c-dvcg.github.io/lds-ed25519-2018/}
+   * for
+   * [jsonld-signatures.]{@link https://github.com/digitalbazaar/jsonld-signatures}
+   * @description Creates an [Ed25519]{@link https://ed25519.cr.yp.to/}
+   * Key Pair usable by a json-ld signature.
+   * @example
+   * const ops = {
+   *   publicKeyBase58: 'H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV',
+   *   privateKeyBase58: 'K3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV'
+   * };
+   * const EDKey = new Ed25519KeyPair(ops);
+   * @param {KeyPairOptions} options - Keys must be in Base58
+   * other options most follow
+   * [KeyPairOptions]{@link ./index.md#KeyPairOptions}.
+   * @param {string} options.publicKeyBase58 - Public Key exposed in Signatures.
+   * @param {string} options.privateKeyBase58 - Private Key for signing.
    */
   constructor(options = {}) {
     super(options);
     this.type = 'Ed25519VerificationKey2018';
     this.privateKeyBase58 = options.privateKeyBase58;
     this.publicKeyBase58 = options.publicKeyBase58;
   }
-
+  /**
+   * @description Returns the public key.
+   * @example
+   *   edKeyPair.publicKey;
```

Comment:
The output of these examples is weird.  Should just make text flush left.  Maybe also look at how [array-tools](https://github.com/75lb/array-tools#api-reference) does examples.

Reply:
- [ ] reformat example to use a > at the beginning and the next line represents the output.